### DISCUSSION TITLE
How can I eliminate forward slashes before apostrophes?

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1072] [General]
I've been working on getting the forms correct (see #823) and whenever there is an apostrophe, it throws in a forward slash such as `Women\'s Big Book Study` or `St. Peter\'s Church`. Is there a quick way to globally strip these slashes out?

**Comment by joshreisner** [comment]
need some more context - are you asking about for example the blue area on [this page](https://sandbox.sussexaa.org/meeting-change/?meeting=womens-big-book-study&id=8469&meetingName=Women%27s+Big+Book+Study&meetingDay=5&time=17%3A00&end_time=18%3A00&meetingTypes=B%2CO%2CW&notes=&conference_url=&conference_url_notes=&conference_phone=&conference_phone_notes=&location=St+Martha%27s+Episcopal+Church&formatted_address=117+Maplewood+St%2C+Bethany+Beach%2C+DE+19930%2C+USA&region=Bethany+Beach&location_notes=&group=Women%27s+Big+Book+Study+Group&group_notes=&district=District+13&website=&email=&phone=&venmo=&square=&paypal=)?

is this a custom PHP page? if so maybe try the `stripslashes` function? https://www.php.net/manual/en/function.stripslashes.php

**Reply by joshreisner** [reply]
i'm not seeing slashes in the URL, which is coming from TSML, so i'm guessing PHP is adding them when getting the meeting name from the query string. eg `echo stripslashes($_GET['meetingName']);`

**Reply by Grateful-Viv** [reply]
‘are you asking about for example the blue area on [this page](https://sandbox.sussexaa.org/meeting-change/?meeting=womens-big-book-study&id=8469&meetingName=Women%27s+Big+Book+Study&meetingDay=5&time=17%3A00&end_time=18%3A00&meetingTypes=B%2CO%2CW&notes=&conference_url=&conference_url_notes=&conference_phone=&conference_phone_notes=&location=St+Martha%27s+Episcopal+Church&formatted_address=117+Maplewood+St%2C+Bethany+Beach%2C+DE+19930%2C+USA&region=Bethany+Beach&location_notes=&group=Women%27s+Big+Book+Study+Group&group_notes=&district=District+13&website=&email=&phone=&venmo=&square=&paypal=)?’ 
yes. 

‘ i'm not seeing slashes in the URL, which is coming from TSML, so i'm guessing PHP is adding them when getting the meeting name from the query string. eg echo stripslashes($_GET['meetingName']);’
I wasn’t sure how to get the URL, so I made it by combining https://sandbox.sussexaa.org/meetings/slug

Is there something I can add to the functions.php to globally strip this?

**Reply by joshreisner** [reply]
the slashes are only showing up in this blue box right? was this not custom PHP code you added? is it gravity forms? 

perhaps if we could see a screenshot of where you composed that box we an make a recommendation

**Reply by Grateful-Viv** [reply]
It is custom PHP code to use with Gravity Forms, who helped me with writing it. I am using Code Snippets since it's easier to make minor changes. If you have that, I can export the snippet. Or I can just put it in a text file, which I hope I can attach here. 

Notes: Form 11 is the ID of the form I am using and Field 75 is the html field to display what you see in the blue box,

[Replace-content-HTML-field-with-content-from-GET.txt](https://github.com/code4recovery/12-step-meeting-list/files/10713671/Replace-content-HTML-field-with-content-from-GET.txt)

**Reply by joshreisner** [reply]
thanks, yes i think the stripslashes function should work for you. try replacing
```
	$meetingName = rgget( 'meetingName' );
```
with
```
	$meetingName = stripslashes(rgget( 'meetingName' ));
```

**Reply by Grateful-Viv** [reply]
Do the same thing for location? 

And do I need to add the striplashes function, or is it already part of the php and I just need to call it, like in your example.

**Reply by joshreisner** [reply]
it's a built-in PHP function - more info here: https://www.php.net/manual/en/function.stripslashes.php

yes you should use it for any variable you need it for

**Reply by Grateful-Viv** [reply]
It worked! Thank you.
